### DISCUSSION TITLE
Select multiple automation parameters at once?

### DISCUSSION THREAD

**Discussion by dsponaugle** [apohl79/audiogridder#366] [Ideas]
Is it possible to select multiple automation parameters in the plugin without having to right-click on the plugin within AG every time to add an additional parameter? Maybe there's a hot-key I'm not aware of but it would be handy if you know you're going to be selecting 16 or so parameters and could do it holding down cmd or something like that to prevent the automation screen from disappearing after a single automation parameter is selected.

**Comment by apohl79** [comment]
Moved this to ideas, as there is no such functionality yet.
